docs/userguide/containers/dockervolumes.md

```diff
@@ -159,6 +159,44 @@ user with access to host and its mounted directory.
 >should be portable. A host directory wouldn't be available on all potential
 >hosts.
 
+### Mount a shared-storage volume as a data volume
+
+In addition to mounting a host directory to your container, Docker
+enables [volume plugins](../../extend/plugins_volume.md) which allow
+provisioning and mounting of an iSCSI, NFS, or FC volumes. The difference
+with volume plugins is that the data volume is automatically provisioned
+or made available from externally shared storage and mounted to the 
+Docker host where your container runs.
+
+>**Note**: A benefit of using shared volumes is that they are _NOT_ 
+>host-dependent. This means that the volume can be made available to
+>any host that a container is started on as long as it has access to
+> the shared storage backend.
+
+One way to use volume drivers is through the `docker run` command. 
+Volume drivers create volumes by name, instead of by path like in
+the other examples.
+
+```
+$ docker run -d --volume-driver=flocker -P --name web -v my-named-volume:/opt/webapp training/webapp python app.py
+```
+
+Here we've mounted a volume named "my-named-volume" that was provisioned
+by a volume driver and made available within the container at `/opt/webapp`.
+
+>**Note**: As of Docker 1.9+ you may also use the `docker volume create` command.
```

Comment:
In general we don't mention which version something is added in (unless really necessary); it's already mentioned in the changelog, and those notes are outdated pretty fast.